### ISSUE TITLE
Bug. AttributeError: 'booz_xform._booz_xform.Booz_xform' object has no attribute 'pmns_b'

### ISSUE THREAD

**Issue by panosz** [hiddenSymmetries/booz_xform#4]
Hi.

When I try to use the wireplot function from the plots module I get the exception in the title.
I use python3.8 in ubuntu20.04

Below is a minimal script to reproduce this. Note that the file path is set assuming that the script is saved and run from the `examples` subfolder :

```python
import booz_xform as bx

b = bx.Booz_xform()
b.read_wout('../tests/test_files/wout_LandremanSenguptaPlunk_section5p3.nc')

b.run()

fig = bx.wireplot(b, js=4)

fig.show()
```

**Comment by panosz** [comment]
Could this be due to d6882133762312aefa7c3cfd50ec081adf2c8998 ?

**Comment by landreman** [comment]
Resolved by 821f4a0693a59756d91b1693d0485409d97bbedc
